python/gui/auto_generated/qgsformlabelformatwidget.sip.in

```diff
@@ -0,0 +1,57 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/gui/qgsformlabelformatwidget.h                                   *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
+ ************************************************************************/
+
+
+
+class QgsFormLabelFormatWidget : QWidget
```

Comment:
Does this one need to be exposed to sip?

Reply:
No strong opinion: I can removed it from sip.